### PR TITLE
Add all-state-authorities handler

### PR DESCRIPTION
**Pull request by cap10morgan**
[Pivotal card](https://www.pivotaltracker.com/story/show/153681289)

This is needed to return every state's voter registration method in a single API call as the linked card calls for.

### CODE REVIEW

File: src/election_authority_works/us_states.clj [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+   "WI"
+   "WY"])
+
+(def ocd-ids
+  (reduce (fn [ids s]
+            (conj ids
+                  (->> s
+                       str/lower-case
+                       (str us-ocd-id "/state:"))))
+          [] postal-abbreviations))
```
Comment (theJohnnyBrown) [inline src/election_authority_works/us_states.clj]: That's a funny way to spell `map`

File: src/election_authority_works/core.clj [hunk 2]
```diff
@@ -10,5 +10,4 @@
     (.addShutdownHook
      (Runtime/getRuntime)
      (Thread. (fn []
-                (queue/close-all! rabbit-resources)
-                (queue/close-all!))))))
+                (queue/close-all! rabbit-resources))))))
```
Comment (theJohnnyBrown) [inline src/election_authority_works/core.clj]: Is this change related to the purpose of the PR? It's unclear why that `close-all!` call was removed

Comment (cap10morgan) [reply src/election_authority_works/core.clj]: No it isn't. This was a target of opportunity b/c that call was doing nothing. Must have been an old typo or something.

File: src/election_authority_works/us_states.clj (hunk 1 above)
Comment (cap10morgan) [reply src/election_authority_works/us_states.clj]: Ah, yep, that would be better here.
